cfme/tests/candu/test_candu_manual.py

```diff
@@ -540,3 +540,32 @@ def test_ec2_instance_memory_metrics():
                 7. Memory metrics should have data.
     """
     pass
+
+
+@pytest.mark.manual
```

Comment:
Since all of the tests in this file are manual, I'd suggest moving all manual marker decorators into a pytestmark at the module level.

Reply:
Yes @tpapaioa , sounds good, will move these decorators at module level.